app/schemas/com.jerboa.db.AppDB/21.json

```diff
@@ -1,218 +0,0 @@
-{
-  "formatVersion": 1,
```

Comment:
Why is this file being removed?

Reply:
Also add your 22 schema

Reply:
I have gone through so many schema migrations with other PRs being merged I'm not sure which should be included or not...